.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+
+# Contributing to NEO
+Neo is an open-source project and it depends on its contributors and constant community feedback to implement the features required for a smart economy. You are more than welcome to join us in the development of Neo.  
+
+Read this document to understand how issues are organized and how you can start contributing.
+
+*This document covers this repository only and does not include community repositories or repositories managed by NGD Shanghai and NGD Seattle.*
+
+### Questions and Support
+The issue list is reserved exclusively for bug reports and features discussions. If you have questions or need support, please visit us in our [Discord](https://discord.io/neo) server.  
```

Comment:
The issue list is reserved exclusively for bug reports and features discussions. If you have questions or need support, please visit us in our [Discord](https://discord.io/neo) server. 

I dont agree with this, for sure and make things, I already modified this subtle sometime ago.
This should be changed to:

"The issue list is open for any kind of discussion, question and interaction with the collaborators of the project. In addition, it is also a good place for bug reports and features discussions. Do not hesitate to open any discussion."

Reply:
@vncoelho  This file was just moved, it does not relate to this PR. Do you want me to put the file back in the root folder?

Reply:
I do not know how it was spruced. This should be fixed asap

Reply:
@vncoelho  no problem, I updated it, can you check?

Reply:
Checked. Looks good.